cmd/queue/main.go

```diff
@@ -129,7 +129,10 @@ func initEnv() {
 
 func reportStats(statChan chan *autoscaler.Stat) {
 	for {
```

Comment:
Wouldn't

`for s := range statChan {
...`

be better?

Reply:
It would.